packages/features/rudder_plugin_db_encryption/ios/rudder_plugin_db_encryption.podspec

```diff
@@ -0,0 +1,26 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint rudder_plugin_db_encryption.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'rudder_plugin_db_encryption'
+  s.version          = '1.0.0'
+  s.summary          = 'iOS Module of the RudderStack SDK for Flutter, to enable database encryption.'
+  s.description      = <<-DESC
+                        This module enables database encryption for the RudderStack SDK for Flutter.
+                       DESC
+  s.homepage         = 'https://github.com/rudderlabs/rudder-sdk-flutter'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'RudderStack' => 'sdk@rudderstack.com' }
```

Comment:
```suggestion
  s.author           = { 'RudderStack' => 'sdk-accounts@rudderstack.com' }
```

Reply:
Changed the email address across all the podspec files.